e2e/content-services/metadata/metadata-properties.e2e.ts

```diff
@@ -34,13 +34,9 @@ describe('CardView Component - properties', () => {
     const METADATA = {
```

Comment:
Same here - we use only PROPERTY_TAB so whole object seems redundant.

Reply:
@DenysVuika can't we remove rest fields as well? Only PROPERTY_TAB field is used